src/booleans.js

```diff
@@ -1,63 +1,123 @@
 function negate(a) {
-  // your code here
+  return !a;
 };
 
 function both(a, b) {
-  // your code here
+  if (a === true && b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function either(a, b) {
-  // your code here
+  if (a === true || b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function none(a, b) {
-  // your code here
+  if (a !== true && b !== true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function one(a, b) {
-  // your code here
+  if (a === true && b === false || a === false && b === true) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function truthiness(a) {
-  // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
-  // your code here
+  if (a === b) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  if (a > b) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  if (a <= b) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isOdd(a) {
-  // your code here
+  if (a%2 !== 0) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isEven(a) {
-  // your code here
+  if (a%2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function isSquare(a) {
-  // your code here
+  if (Math.sqrt(a) % 1 === 0) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function startsWith(char, string) {
-  // your code here
+  if (string[0] === char) {
+    return true;
+  } else {
+    return false;
+  }
 };
 
 function containsVowels(string) {
-  // your code here
+let vowels = 'aeiouAEIOU';
```

Comment:
Would definitely be better to use the regex alternative here, though I think what you have in the comment isnt the right way to do it. Refer to your solution for `elementsStartingWithAVowel` above (and be careful about the use of the `^` character...)